testing/nist-sts/0001-Solve-the-memory-out-of-bounds-problem-in-sts.patch

```diff
@@ -0,0 +1,27 @@
+From 530b8e0d0f9ea68f1dd977caddc76fbe4a2c66b8 Mon Sep 17 00:00:00 2001
+From: makejian <makejian@xiaomi.com>
+Date: Wed, 19 Apr 2023 22:57:44 +0800
+Subject: [PATCH 2/2] Solve the memory out of bounds problem in sts
+
+Signed-off-by: makejian <makejian@xiaomi.com>
+Change-Id: I28a132a793603205e25e5925de3e51ff2c44873b
+---
+ sts/src/discreteFourierTransform.c | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/sts/src/discreteFourierTransform.c b/sts/src/discreteFourierTransform.c
```

Comment:
it need add --dst-prefix= and --src-prefix= to git format-patch, please reference other 3rd party project.